docker/caddy/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:12.17.0 AS client-builder
+
+COPY src ./src
+COPY static ./static
+COPY gatsby-config.js .
+COPY package.json .
+COPY yarn.lock .
+
+RUN yarn --frozen-lockfile
+RUN yarn build
```

Comment:
Yeah I wanted to avoid that because when running that locally, there is bunch of stuff that we don't want to copy over like node_modules or cache directory. They can get pretty huge and it slows down the build by a lot.

Reply:
OK, that makes sense.